starter-code/routes/celebrities.routes.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express')
+const router = express.Router()
+const CelebrityModel = require('../models/celebrity')
+
+
+router.get('/celebrities', (req, res, next) => {
+    CelebrityModel.find()
+        .then((celebrity)=>{
+            // console.log(celebrity)
+            res.render('./celebrities/index.hbs', {celebrity})
+        })
+        .catch((err)=>{
+           console.log('Error is:', err)
+           next()
+        })
+})
+
+
+router.get('/celebrities/new', (req, res, next)=>{
+    res.render('./celebrities/new.hbs')
+})
+
+router.get('/celebrities/:id/edit',(req, res, next)=>{
+    CelebrityModel.findById(req.params.id)
+    .then((celeb)=>{
+        console.log('Data added')
+        res.render('celebrities/edit.hbs', {celeb})
+    })
+    .catch ((err)=>{
+        next()
+        console.log(err)
+    })
+})
+
+router.post('/celebrities/new', (req, res)=>{
+    CelebrityModel.create(req.body)
+        .then(()=>{
+            console.log('Data added')
+            res.redirect('/celebrities')
+        })
+        .catch (()=>{
+            res.render('/celebrities/new.hbs')

```

Comment:
Two issues here.
1. The render won't work because of the / at the beginning. Remember renders should never start with a /.
2. a redirect would make more sense here. since you already have a route that is taking care of rendering that page. (/celebrities/new)